pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -150,3 +150,160 @@ func createMemEnterScopeHinter(resolver hintReferenceResolver, memset bool) (hin
+			//>		f'Invalid value for elm_size. Got: {elm_size}.'
+			//> key = ids.key
+			//>
+			//> if '__find_element_index' in globals():
+			//>		ids.index = __find_element_index
+			//>		found_key = memory[array_ptr + elm_size * __find_element_index]
+			//>		assert found_key == key, \
+			//>			f'Invalid index found in __find_element_index. index: {__find_element_index}, ' \
+			//>			f'expected key {key}, found key: {found_key}.'
+			//>		# Delete __find_element_index to make sure it's not used for the next calls.
+			//>		del __find_element_index
+			//> else:
+			//>		n_elms = ids.n_elms
+			//>		assert isinstance(n_elms, int) and n_elms >= 0, \
+			//>			f'Invalid value for n_elms. Got: {n_elms}.'
+			//>		if '__find_element_max_size' in globals():
+			//>			assert n_elms <= __find_element_max_size, \
+			//>				f'find_element() can only be used with n_elms<={__find_element_max_size}. ' \
+			//>				f'Got: n_elms={n_elms}.'
+			//>
+			//>		for i in range(n_elms):
+			//>			if memory[array_ptr + elm_size * i] == key:
+			//>				ids.index = i
+			//>				break
+			//>		else:
+			//>			raise ValueError(f'Key {key} was not found.')
+			arrayPtrAddr, err := hinter.ResolveAsAddress(vm, arrayPtr)
+			if err != nil {
+				return err
+			}
+			elmSizeVal, err := hinter.ResolveAsUint64(vm, elmSize)
+			if err != nil {
+				return err
+			}
+			if elmSizeVal == 0 {
+				return fmt.Errorf("Invalid value for elm_size. Got: %v", elmSizeVal)
+			}
+			keyVal, err := hinter.ResolveAsFelt(vm, key)
+			if err != nil {
+				return err
+			}
+			findElementIndex, err := ctx.ScopeManager.GetVariableValue("__find_element_index")
+			if err == nil {
+				findElementIndex := findElementIndex.(uint64)
+				findElementIndexFelt := new(fp.Element).SetUint64(findElementIndex)
+				findElementIndexMemoryValue := memory.MemoryValueFromFieldElement(findElementIndexFelt)
+				indexValAddr, err := index.GetAddress(vm)
+				if err != nil {
+					return err
+				}
+				err = vm.Memory.WriteToAddress(&indexValAddr, &findElementIndexMemoryValue)
+				if err != nil {
+					return err
+				}
+				arrayPtrAddr.Offset = arrayPtrAddr.Offset + elmSizeVal*findElementIndex
+				foundKey, err := vm.Memory.ReadFromAddress(arrayPtrAddr)
+				if err != nil {
+					return err
+				}
+				foundKeyVal, err := foundKey.FieldElement()
+				if err != nil {
+					return err
+				}
+				if foundKeyVal.Cmp(keyVal) != 0 {
+					return fmt.Errorf("Invalid index found in __find_element_index. index: %v, expected key %v, found key: %v", findElementIndex, keyVal, &foundKey)
+				}
+				err = ctx.ScopeManager.DeleteVariable("__find_element_index")
+				if err != nil {
+					return err
+				}
+			} else {
+				nElms, err := hinter.ResolveAsUint64(vm, nElms)
+				if err != nil {
+					return err
+				}
+				findElementMaxSize, err := ctx.ScopeManager.GetVariableValue("__find_element_max_size")
+				if err == nil {
+					findElementMaxSize := findElementMaxSize.(uint64)
```

Comment:
i would check for `!ok`

Reply:
I think you forgot to fix this 😛

Reply:
The same thing happend in other place, where I changed that so mb haha 😜, fixed this as well